jmeter-java-dsl/src/test/java/org/apache/jmeter/gui/util/MenuFactory.java

```diff
@@ -105,80 +106,95 @@ public final class MenuFactory {
 
   private static Set<String> classesToSkip() {
     return Arrays.stream(JMeterUtils.getPropDefault("not_in_menu", "").split(","))
-        .map(String::trim)
-        .collect(Collectors.toSet());
+            .map(String::trim)
```

Comment:
If you can please try to align the IDE code style to the project, or only submit changes that are not related with spacing to avoid back and forth of changes regarding this and also focus PR in the relevant changes.